collections_hierarchy/mixins.py

```diff
@@ -1,53 +1,105 @@
 class ComparableMixin(object):
     def __eq__(self, other):
-        pass
+        if self.data == other.data:
```

Comment:
Santiago mentioned this to us, but when we did self.data, we were hard coding it vs passing it in dynamically. Ask him further about it you have questions!